README.md

```diff
@@ -40,35 +39,43 @@ Parse Server works with the Express web application framework. It can be added t
 
 # Getting Started
 
-April 2016 - We created a series of video screencasts, please check them out here: [http://blog.parseplatform.org/learn/parse-server-video-series-april-2016/](http://blog.parseplatform.org/learn/parse-server-video-series-april-2016/)
+Parse-Server is built on the top of node. Before you can get started, you should have the latest version of node and npm installed.
+We always recommend you run your server on the LTS versions as we focus on providing the most stability on those versions.
 
-The fastest and easiest way to get started is to run MongoDB and Parse Server locally.
+You can find the latest versions of [node.js here](https://nodejs.org/en/).
 
-## Running Parse Server
+In order to be fully functional, `parse-server` requires a database to run. You'll need to either have [mongodb](https://www.mongodb.com/download-center) or [postgres](https://www.postgresql.org/download/) setup, locally or remotely.
 
-### Locally
-```
-$ npm install -g parse-server mongodb-runner
-$ mongodb-runner start
-$ parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://localhost/test
-```
-***Note:*** *If installation with* `-g` *fails due to permission problems* (`npm ERR! code 'EACCES'`), *please refer to [this link](https://docs.npmjs.com/getting-started/fixing-npm-permissions).*
- 
+There are many service providers that offer hosting of mongodb and postgres.
+
+:warning: Before creating your first app, your DB should be accessible and running.
+
+## Creating your first application
 
-### Inside a Docker container
+In order to create your first application, you can use the `@parse/init` project.
+
+The `@parse/init` project is an interactive command line tool that bootstraps a functional parse-server setup on your local machine.
+
+In order to create your first parse app in the `my-parse-app` folder, run the following command:
```

Comment:
This is the first mention of `my-parse-app`  is this an empty directory?  Do I make it, or does it exist somewhere?

Reply:
It creates it, if the folder exists, it aborts on it’s own with a relevant error message